### ISSUE TITLE
Bancor TVL issue

### ISSUE THREAD

**Issue by ishraq8** [messari/subgraphs#1109]
Bancor team brought up that there is a difference in TVL compared to our subgraph

Etherscan of V3 vault: https://etherscan.io/tokenholdings?a=0x649765821D9f64198c905eC0B2B037a4a52Bc373 

I suspect the issue is that we are measuring their deposits (they are single sided) and our subgraph doesn't reflect the current deficits in the pools (this was noted here in the original QA: https://docs.google.com/spreadsheets/d/1Yg4JvsYiCccmQhhVwUKvfWffXSB165GyqaVbvGRc82o/edit#gid=0). Will respond to them and raise issues if needed.

**Comment by ishraq8** [comment]
@steegecs so we talked to Bancor. the current pool and TVL is based on the staked amount. we should change to amount in the pools. the staked amount is different from the pool amount. 

https://etherscan.io/tokenholdings?a=0x649765821D9f64198c905eC0B2B037a4a52Bc373

would it be possible to change this? i can do a quick QA after to make sure it is good. not a huge deal, we had documented this issue before and it seems the bancor team recommends doing amounts in pool vs staked amount.

**Comment by bye43** [comment]
@ishraq8 Bumping this. Is it still relevant?

**Comment by ishraq8** [comment]
@bye43 Yeah, this got backlogged. Bancor asked us to calculate their pool TVL and amount by how much is there and not how much was deposited. They are facing a deficit and surplus in certain pools. May want to assign someone who is free.
